### ISSUE TITLE
image_enhance attributte not present

### ISSUE THREAD

**Issue by Ankit12063** [ivonajdenkoska/fingerprint-recognition#5]
![Screenshot (300)](https://user-images.githubusercontent.com/71066451/108836959-8691b500-75f7-11eb-8147-15c30cc9593f.png)
it's saying module 'fprmodules.enhancement' has no attribute 'image_enhance'

**Comment by KaiWangCanada** [comment]
same issue

**Comment by SuryaRam1592** [comment]
Same Issue

**Comment by tawhidwasik08** [comment]
So, here is the issue .. this repo uses [this](https://github.com/Utkarsh-Deshmukh/Fingerprint-Enhancement-Python) repo for enhancing the fingerprint. As it is suggested, you have to clone the repo in **src/fprmodules/enhancement** folder. But the problem is I believe enhancement repo is since been updated so that it doesn't have function **image_enhance**.

You can try the following:

1. Copy FingerprintImageEnhancer.py from src/ folder of the enhancement repo to src/fprmodules/enhancement/ folder
2. In utils.py add `from fprmodules.enhancement.FingerprintImageEnhancer import FingerprintImageEnhancer`
3. In utils.py rewrite `def enhance_image():` as <br/>
``` 
def enhance_image(image):
    image_enhancer = FingerprintImageEnhancer() 
    img_e = image_enhancer.enhance(image)
    img_e = np.asarray(img_e, dtype="uint8")
    return cv2.normalize(img_e, None, alpha=0, beta=255, norm_type=cv2.NORM_MINMAX, dtype=0)
```

Though right now I am seeing some decrease in performance as well as seeing different outcome for enhanced_image.

**Comment by Ankit12063** [comment]
Thanks but i have already solved this problem

**Comment by tawhidwasik08** [comment]
Nice !
Did you use the mentioned dataset ? If yes, could you kindly tell me how much accuracy you are getting in identification and verification scenarios ?

**Comment by Ankit12063** [comment]
Yes, i am using the same dataset. You can check my github profile for
accuracy
